release_notes/Service_Fabric_ReleaseNotes_81CU3.md

```diff
@@ -0,0 +1,80 @@
+* [Repositories and Download Links](#repositories-and-download-links)
+
+
+## Current Breaking Changes
+* Service Fabric 7.2 and higher runtime drops support for .NET Core Service Fabric apps running with .NET Core 2.2 runtime. .NET Core runtime 2.2 is out of support from Dec 2019. Service Fabric runtime will not install .NET Core runtime 2.2 as part of its dependency. Customers should upgrade their .NET 2.2 runtime SF apps to the next .NET Core LTS version 3.1.<br>
+* Guest executable and container applications created or upgraded in SF clusters with runtime versions 7.1+ are incompatible with prior SF runtime versions (e.g. SF 7.0).<br/>
+    Following scenarios are impacted:<br/>
+    * An application with guest executables or containers is created or upgraded in an SF 7.1+ cluster.<br/>
+    The cluster is then downgraded to a previous SF runtime version (e.g. SF 7.0).<br/>
+    The application fails to activate.<br/>
+    * A cluster upgrade from pre-SF 7.1 version to SF 7.1+ version is in progress.<br/>
+    In parallel with the SF runtime upgrade, an application with guest executables or containers is created or upgraded.<br/>
+    The SF runtime upgrade starts rolling back (due to any reason) to the pre-SF 7.1 version.<br/>
+    The application fails to activate.<br/>
+
+    To avoid issues when upgrading from a pre-SF 7.1 runtime version to an SF 7.1+ runtime version, do not create or upgrade applications with guest executables or containers while the SF runtime upgrade is in progress.<br/>
+    * The simplest mitigation, when possible, is to delete and recreate the application in SF 7.0.<br/>
+    * The other option is to upgrade the application in SF 7.0 (for example, with a version only change).<br/>
+
+    If the application is stuck in rollback, the rollback has to be first completed before the application can be upgraded again.
+* .NET Core runtime LTS 2.1 runtime is now out of support as of Aug 21, 2021. Service Fabric releases after that date will drop support for Service Fabric apps running with .NET Core 2.1 runtime. This includes the following release versions and beyond - 8.0 CU4 and above, 8.1 CU2 and above and 8.2 RTO release and above. Current apps running on .NET Core 2.1 runtime will continue to work, but requests for investigations or request for changes will no longer receive support. Service Fabric .NET SDK will take a dependency on .NET runtime 3.x features to support Service Fabric .NET Core apps. This has no impact on Service Fabric .NET Framework SDK.
+* Ubuntu 16.04 LTS reached it's 5-year end-of-life window on April 30, 2021 and is no longer supported by it's vendors. Service Fabric runtime will drop support for Ubuntu 16.04 after that date as well. Current applications running on it will continue to work, but requests for investigations or requests for change will no longer receive support. We recommend moving your applications to Ubuntu 18.04.
```

Comment:
Our https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-versions#supported-linux-versions-and-support-end-date page does not align with this. Needs alignment. May be helpful to also generalize the recommendation to "a supported linux distribution as documented at <url>" in future since when 20.04 support goes live this becomes stale info.

Reply:
I have an action item for updating the versions page when we publish the 8.2 RTO release notes. I will be aligning it at that time.
Regarding the 20.04 release - we don't plan for that till 2022, so I don't want to generalize it just yet. But, your feedback is good. Please do re-surface this feedback when we are publishing the 9.0 RTO release in March-2022 which should contain the Ubuntu 20.04 support public release.